src/features/projectsV2/ProjectList/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { useTranslations } from 'next-intl';
+import { useCallback, useMemo } from 'react';
+import styles from '../ProjectsSection.module.scss';
+import NoProjectFound from '../../../../public/assets/images/icons/projectV2/NoProjectFound';
+import { useProjects } from '../ProjectsContext';
+import ProjectSnippet from '../ProjectSnippet';
+import { MapProject } from '../../common/types/projectv2';
+
+const ProjectList = () => {
+  const tAllProjects = useTranslations('AllProjects');
+  const {
+    debouncedSearchValue,
+    selectedClassification,
+    filteredProjects,
+    tabSelected,
+    topProjects,
+    projects,
+  } = useProjects();
+  const ProjectsToDisplay = useMemo(() => {
+    const hasClassificationOrSearch =
+      debouncedSearchValue !== '' || selectedClassification.length > 0;
+    if (hasClassificationOrSearch) return filteredProjects;
+    return tabSelected === 'topProjects' ? topProjects : projects;
+  }, [filteredProjects, tabSelected]);
+
+  const sortedProjects = useMemo(() => {
+    return ProjectsToDisplay?.sort((a, b) => {
+      return (
+        Number(b.properties.allowDonations) -
+        Number(a.properties.allowDonations)
+      );
+    });
```

Comment:
The sorting logic could be changed as below, to make it clear that projects that allow donations are sorted at the top. 

```
const sortedProjects = useMemo(() => {
  return ProjectsToDisplay?.sort((a, b) => {
    if (a.properties.allowDonations === b.properties.allowDonations) return 0;
    return a.properties.allowDonations ? -1 : 1;
  });
}, [ProjectsToDisplay]);
```